source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -448,8 +447,8 @@ def _set_specific_params(mod):
                 args['_max_successive_physical_mode'] = mod.max_successive_physical_mode
             if args.get('_final_line_filter') is None:
                 args['_final_line_filter'] = mod.final_line_filter
-            if args.get('_max_extra_second_pass') is None:
-                args['_max_extra_second_pass'] = mod.max_extra_second_pass
+            if args.get('max_extra_second_pass') is None:
+                args['max_extra_second_pass'] = mod.max_extra_second_pass
             if args.get('additional_time_after_first_section_taxi') is None:
```

Comment:
What about https://github.com/CanalTP/navitia/blob/3a4d48a14deb21e7bc03cf999c35e859d29776a5/source/jormungandr/jormungandr/interfaces/v1/Journeys.py#L271 ?

Don't you have also to modify it to use a parameter 'max_extra_second_pass' ?

Reply:
It's actually where the magic happens (and it was not spotted when the previous code was written).
Here we expose a (hidden) param used in url under the name `_max...` but we store and use it under the name `max...` (with `dest=`). So it's the only place with doc where the `_` needs to be, as we never manage args directly (almost).